test/e2e/README.md

```diff
@@ -174,6 +180,17 @@ func init() {
 	os.Setenv("E2E_NODE", "validator01")
 }
 ```
+### Saving the runner executions
+
+If you need to save the files used in a runner execution, specify the `--execution` or `-e` flag, which will save the
+node files, manifest, and any other asset used during the test. It will also save the docker log files for each node.
+Each execution is saved under a `network_executions` folder, and at each run, a subfolder with a time and date timestamp
+will be created as the name of the folder. This way, you can test changing parameters in the manifest and later check
+the logs to see how that affected the run behavior and tests.
+
+```
+./build/runner -f networks/simple --execution
+```
```

Comment:
I would add here a comment mentioning that the saved executions can get pretty large in some cases, so to use this with care.

Reply:
This is specially relevant to the `data` dirs of each CometBFT home directory, but it can also be the case of logs, in particular when `debug` level is used.